redaxo/src/addons/debug/boot.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 // collect only data in debug mode with http requests outside of the debug addon
-if (!rex::isDebugMode() || !rex_server('REQUEST_URI') || 'debug' === rex_get(rex_api_function::REQ_CALL_PARAM)) {
+if (!rex::isDebugMode() || !(rex_server('REQUEST_URI') || rex::getConsole()) || 'debug' === rex_get(rex_api_function::REQ_CALL_PARAM)) {
```

Comment:
Diese Bedingungen liest sich sehr schlecht IMO

Reply:
Ich finde vor allem negierte Klammerausdrücke immer schwierig (`!(... ...)`). Daher wäre das hier für mich bereits besser lesbar:

```suggestion
if (!rex::isDebugMode() || !rex_server('REQUEST_URI') && !rex::getConsole() || 'debug' === rex_get(rex_api_function::REQ_CALL_PARAM)) {
```

Wobei ich mich frage, was fällt denn überhaupt unter `!rex_server('REQUEST_URI') && !rex::getConsole()`?
Ging es vorher nicht genau darum, die Console auszuschließen, und könnte man somit nicht jetzt beides weglassen?

Reply:
> Wobei ich mich frage, was fällt denn überhaupt unter !rex_server('REQUEST_URI') && !rex::getConsole()?

eigentlich nur noch cli scripte, die nicht über die console laufen. z.B. unsere unit tests oder?

Reply:
ich mach dafür nen getrennten PR los und checke mal, welche Conditions wir eig. da brauchen.